what-is-the-ic/3_canisters/index.md

```diff
@@ -4,16 +4,11 @@ card: /img/what-is-the-ic/canister.jpg
 cardImageFit: cover
 ---
 
-A smart contract is a computer program executed on a blockchain. A *canister*, or canister smart contract, is a bundle comprising a computer program and its data. Every canister is hosted on one subnet of the IC.
+A smart contract is a computer program executed on a blockchain. A canister is a smart contract that bundles a computer program and its data. Every canister is hosted on one subnet of the IC.
+Canisters can be executed concurrently and communicate within and across subnets by sending asynchronous messages in a non-blocking manner, resulting in essentially unbounded scalability.
+Canisters are more powerful than smart contracts on other blockchains as they can
+* serve a user interface directly to any web browser,
+* hold gigabytes of memory for a low fee, perform substantial amounts of computation at a low cost, and pay for their own computation (learn more about the [reverse gas model](https://internetcomputer.org/features/reverse-gas/)).
 
-Canisters on different subnets can be executed concurrently. Furthermore, multiple canisters on the same subnet can also be executed in parallel, further increasing throughput. Canisters communicate within and across subnets by sending asynchronous messages in a non-blocking manner. These properties allow for essentially unbounded scalability.
-
-Canisters on the IC have distinguishing properties. They can
-
-- serve a user interface directly from the blockchain,
-- hold gigabytes of memory for a low fee,
-  perform substantial amounts of computation at a low cost, and
-  pay for their own computation (learn more about the [reverse gas model](https://internetcomputer.org/features/reverse-gas/)).
-
-Engineers can implement canisters in any language that compiles to WebAssembly. SDKs are currently available for [Rust](/docs/current/developer-docs/build/cdks/cdk-rs-dfinity/) and [Motoko](/docs/current/developer-docs/build/cdks/motoko-dfinity/motoko/).
+Canisters can be implemented in any language that compiles to WebAssembly. SDKs are currently available for [Rust](/docs/current/developer-docs/build/cdks/cdk-rs-dfinity/) and [Motoko](/docs/current/developer-docs/build/cdks/motoko-dfinity/motoko/).
```

Comment:
Can we link instead to https://internetcomputer.org/docs/current/developer-docs/build/cdks/? There's more and more languages plopping up and we shouldn't have to update this page with the available languages

Reply:
Are you suggesting to change the last sentence to something like "Learn more about available SDKs here (link)"?

Reply:
Added the link